helm/distributed-idol/resources/content_primary.cfg

```diff
@@ -1,887 +1,55 @@
-XSLTemplates=TRUE
+ServicePort={{ .Values.content.servicePort | int }}
 
-[Server]
+[Server] < "/content/cfg/original.content.cfg" [Server]
+< "/content/cfg/idol_ssl.cfg" [EnableSSL] SSLConfig
 Port={{ .Values.content.aciPort | int }}
 IndexPort={{ .Values.content.indexPort | int }}
-
-Threads=4
-MaxInputString=64000
-XSLTemplates=TRUE
-LogRequestTiming=TRUE
-
-// Disable extra search functionality
-AdvancedSearch=FALSE
-AdvancedCaseSearch=FALSE
-AdvancedPlus=FALSE
-
-MaxResults=10000000
+SearchUncommittedDocuments=True
 MaxDocumentCount={{ .Values.autoscaling.maxDocumentCount | int }}
 
-MaxQueryTerms=250
-//QueryTimeoutInMilliseconds=30000
-
-
-AutnRank=TRUE
-
-AutoDetectLanguagesAtIndex=FALSE
-DisallowAgentWeights=TRUE
-// De-duplicate documents by the following fields e.g. */DREREFERENCE
-KillDuplicates=*/DREREFERENCE
-
-// CSV of tags that define the start / end of an XML document
-DocumentDelimiterCSVs=*/DOCUMENT
+[IndexServer] < "/content/cfg/original.content.cfg" [IndexServer]
+< "/content/cfg/idol_ssl.cfg" [EnableSSL] SSLConfig
 
-// When exporting to XML, create the top level field
-XMLTopLevelExportField=DOCUMENTS
+[AuthorizationRoles] < "/content/cfg/original.content.cfg" [AuthorizationRoles]
 
-DateFormatCSVs=SHORTMONTH#SD+#SYYYY,DD/MM/YYYY,YYYY/MM/DD,YYYY-MM-DD,AUTNDATE
-
-ParametricRefinement=TRUE
-// Reduces memory required for parametric refinement at expense of a=GetTagValues&Restriction=... - use a=getQueryTagValues instead
-ParametricMaxPairsPerDocument=0
-// Optimises MATCH operations on Parametric fields - don't need to make them MatchType as well
-ParametricNumericMapping=TRUE
-
-SplitNumbers=FALSE
-SortReturnAllDocuments=TRUE
-
-// The following will help prevent performance degradation when end users use burdensome wildcards like "s*" or "*s*"
-WildcardMaxTerms=1000
-
-// These fields will be preserved through a document update
-KillDuplicatesPreserveFields=*/AU_TAG_*,*/PHRASE_WEIGHT
-
-// The following control the quality of the query summary results at a slight cost to performance.  They may be altered depending on data corpuses and desired results
-QuerySummaryPlus=TRUE
-QuerySummaryAdvanced=TRUE
-QuerySummaryLength=25
-QuerySummaryTerms=350
-
-// For large IDOL server installations or where the connectors produce IDX files
-// smaller than 50-100Mb this setting should be kept TRUE to optimize indexing
-// performance. Alternatively, the connectors batch size could be increased.
-DelayedSync=TRUE
-MaxSyncDelay=120
-
-NodeTableCompression=lz4
-SearchUncommittedDocuments=true
-
-//--------------------------- Authorization Roles ----------------------------//
-[AuthorizationRoles]
-0=AdminRole
-1=QueryRole
-2=IndexRole
-
-[AdminRole]
-StandardRoles=admin,servicecontrol,query,servicestatus
+[AdminRole] < "/content/cfg/original.content.cfg" [AdminRole]
 Clients=*
-//SSLIdentities=
-//GSSPrincipals=
 
-[QueryRole]
-StandardRoles=query,servicestatus
+[QueryRole] < "/content/cfg/original.content.cfg" [QueryRole]
 Clients=*
```

Comment:
The changes for resources/content.cfg modified the Role sections to
```ini
[XxxRole] < "/content/cfg/content.cfg" [XxxRole]
```
(rather than referencing `original.content.cfg` and dropped the `Clients=*` line) — any reason not to make these two consistent?

Reply:
[e3d4c5d](https://github.com/opentext-idol/idol-containers-toolkit/pull/92/commits/e3d4c5d46d2f4602f189348a6ea239c85dcd46cf)